vasyl_varyshchuk/Week_3/task_2/rest_api_requests.py

```diff
@@ -0,0 +1,95 @@
+"""
+This script contains functions for display, create, update and delete users from server with REST API
+"""
+import requests
+import json
+
+URL = 'http://127.0.0.1:5000/api/'
+headers = {'Content-type': 'application/json'}
+
+
+def get_list_of_users():
+    """This function returns list of users from a server"""
+    try:
+        response = requests.get(URL+'users/')
+        response.raise_for_status()
+        return response.json()['users']
+    except requests.RequestException as error:
```

Comment:
Isn't it possible to handle exact errors?

Reply:
Ok, I changed exceptions handling in my code.